spec/system/admin/orders_spec.rb

```diff
@@ -122,6 +122,30 @@
       end
     end
 
+    context "bulk actions" do
+      before do
+        login_as_admin_and_visit spree.admin_orders_path
+      end
+
+      it "can resend confirmation email" do
```

Comment:
We should add a test to verify the unauthorized case.

Reply:
Yes. But I don't think it could be a system spec, as the interface doesn't allow to view/edit orders that aren't allowed to edit.
I'm wondering how we can test Reflexes then.

Reply:
Could this be places to start?
- https://docs.stimulusreflex.com/appendices/testing
- https://github.com/podia/stimulus_reflex_testing

Reply:
> the interface doesn't allow to view/edit orders that aren't allowed to edit.

You can load the page while you are allowed to edit and then revoke that permission before submitting the form.